sklearn/preprocessing/_encoders.py

```diff
@@ -350,6 +363,27 @@ def _compute_drop_idx(self):
                                             len(self.drop)))
             missing_drops = [(i, val) for i, val in enumerate(self.drop)
                              if val not in self.categories_[i]]
+
+            missing_drops = []
```

Comment:
Doesn't the missing_drops implementation from two lines above still work? And if not you should at least remove it? but set operations should work on NaN, right?

Reply:
this hasn't been addressed, right? There is still an implementation two lines above that is then overwritten. And I am pretty sure that the old code still works. you can also use list.index which works on NaN as expected.

Reply:
Since `self.categories_[i]` is a numpy array, I was trying to avoid converting it into another data structure and making a copy.